react-native-quick-crypto.podspec

```diff
@@ -37,7 +38,7 @@ Pod::Spec.new do |s|
     "HEADER_SEARCH_PATHS" => "\"$(PODS_ROOT)/boost\" \"$(PODS_ROOT)/boost-for-react-native\" \"$(PODS_ROOT)/glog\"  \"${PODS_ROOT}/Headers/Public/React-hermes\" \"${PODS_ROOT}/Headers/Public/hermes-engine\""
   }
 
-  s.dependency "OpenSSL-Universal"
```

Comment:
comment about removing the user project modifications from podspec:    

```diff
- "CLANG_CXX_LANGUAGE_STANDARD" => "c++17",
```

Reply:
Done.  Added the required spec to a post_install block in the Podfile of example app.